src/components/Header.js

```diff
@@ -0,0 +1,11 @@
+import './Header.css'
+
+const Header = () => {
+    return (
+        <div className="head-container">
+            <img  className="image" src="https://upload.wikimedia.org/wikipedia/commons/thumb/4/4f/Marvel%27s_The_Avengers_logo.svg/2560px-Marvel%27s_The_Avengers_logo.svg.png" alt ="no-image"/>
```

Comment:
do add a relatable alt text

Reply:
also instead putting this long url here can you create a constants file and push it there